de.metas.adempiere.adempiere/base/src/main/java/de/metas/translation/api/impl/ElementTranslationBL.java

```diff
@@ -0,0 +1,67 @@
+package de.metas.translation.api.impl;
+
+import java.sql.SQLException;
+
+import org.adempiere.ad.trx.api.ITrx;
+import org.compiere.dbPort.Convert;
+import org.compiere.util.DB;
+
+import de.metas.translation.api.IElementTranslationBL;
+
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2017 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+public class ElementTranslationBL implements IElementTranslationBL
+{
+	public static final String AD_Column_TRL_Updater = "update_AD_Column_TRL_On_ElementTrl";
+	public static final String AD_Process_Para_TRL_Updater = "update_AD_Process_Para_TRL_On_ElementTrl";
+	public static final String AD_Field_TRL_Updater = "update_AD_Field_TRL_On_ElementTrl";
+	public static final String AD_PrintFormatItem_TRL_Updater = "update_AD_PrintFormatItem_TRL_On_ElementTrl";
+
+	@Override
+	public void updateTranslations(final int elementId, final String adLanguage) throws SQLException
+	{
+		// Update Columns, Fields, Parameters, Print Info
+		final String trxName = ITrx.TRXNAME_None;
+		{
+			// Column
+			DB.executeFunctionCallEx(trxName, addUpdateFunctionCall(AD_Column_TRL_Updater, elementId, adLanguage), null);
```

Comment:
i would suggest to merge all those db functions into one. I don't see a benefit of having them separated.